bundles/org.openhab.binding.automower/src/main/java/org/openhab/binding/automower/internal/things/AutomowerHandler.java

```diff
@@ -0,0 +1,301 @@
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        if (command instanceof RefreshType) {
+            refreshChannels(channelUID);
+        } else {
+            sendCommand(channelUID, command);
+        }
+    }
+
+    private void sendCommand(ChannelUID channelUID, Command command) {
+        switch (channelUID.getId()) {
+            case CHANNEL_COMMAND_DURATION:
+                if (command instanceof DecimalType) {
+                    commandDuration.set(((DecimalType) command).longValue());
+                    updateState(CHANNEL_COMMAND_DURATION, new DecimalType(commandDuration.get()));
+                }
+                break;
+
+            case CHANNEL_COMMAND:
+                if (command instanceof StringType) {
+                    sendAutomowerCommand(command);
+                } else {
+                    logger.debug("Received command of type {} but StringType is expected.",
+                            command.getClass().getName());
+                }
+                break;
+
+        }
+    }
+
+    private void refreshChannels(ChannelUID channelUID) {
+        updateAutomowerState();
+    }
+
+    @Override
+    public void initialize() {
+        Bridge bridge = getBridge();
+        if (bridge != null) {
+            config = getConfigAs(AutomowerConfiguration.class);
+
+            final String configMowerId = config.getMowerId();
+            if (configMowerId != null) {
+                automowerId.set(configMowerId);
+
+                // bridgeHandler.registerLightStatusListener(this);
+                startAutomowerPolling();
+            } else {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                        "@text/conf-error-no-mower-id");
+            }
+
+        } else {
+
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_UNINITIALIZED);
+        }
+
+    }
+
+    @Nullable
+    private AutomowerBridge getAutomowerBridge() {
+        Bridge bridge = getBridge();
+        if (bridge != null) {
+            ThingHandler handler = bridge.getHandler();
+            if (handler instanceof AutomowerBridgeHandler) {
+                AutomowerBridgeHandler bridgeHandler = (AutomowerBridgeHandler) handler;
+                return bridgeHandler.getAutomowerBridge();
+            }
+        }
+        return null;
+    }
+
+    @Override
+    public void dispose() {
+        logger.debug("Handler disposed.");
+        if (!automowerId.get().equals(NO_ID)) {
+            // bridgeHandler.unregisterLightStatusListener(this);
+            stopAutomowerPolling();
+            automowerId.set(NO_ID);
+        }
+    }
+
+    private void startAutomowerPolling() {
+        if (automowerPollingJob == null || automowerPollingJob.isCancelled()) {
+            if (config.getPollingInterval() < 1) {
+                logger.info("No valid polling interval specified. Using default value: {}s", automowerPollingIntervalS);
```

Comment:
See above

Reply:
ok

Reply:
ok

Reply:
Can you change it? Logging to info should be used rarely. This could be debug.

Reply:
Same question as above: Hm, and how should a user of the plugin be informed about such a misconfiguration? When I use a binding and configure a value that is not used by the binding I definitely would expect that to be logged.
For me this would be a very rare log. It logs one line when the binding is loaded and when it was configured with wrong values.

Reply:
will also use the offline state